tmva/pymva/src/MethodPyKeras.cxx

```diff
@@ -138,6 +140,17 @@ void MethodPyKeras::Init() {
    PyRunString("import sys; sys.argv = ['']", "Set sys.argv failed");
    PyRunString("import keras", "Import Keras failed");
 
+   int num_threads = fNumThreads; 
+   if (num_threads > 0) { 
+      PyRunString("import tensorflow as tf");
+      PyRunString("from keras.backend import tensorflow_backend as K");
+      PyRunString("print 'setting tensorflow to run as single thread' "); 
```

Comment:
This breaks for Theano or CNTK as backend!

Reply:
I agree, I will add an if statement that this will work only in case of Tensorflow